packages/integration-karma/scripts/karma-plugins/env.js

```diff
@@ -30,7 +30,8 @@ function createEnvFile(lwcConfig) {
         `        COMPAT: ${lwcConfig.compat},`,
         `        NATIVE_SHADOW: ${lwcConfig.nativeShadow}`,
         `    }`,
-        `};`,
+        `};
+        if (window.process.env.COMPAT) { delete window.customElements; };`,
```

Comment:
Wiping out the customElements object fixes it for tests. Will the engine work fine in COMPAT mode in google/firefox/safari in non-test modes?

Reply:
hmmm, good question... it will not. For that to work, u need to run the downgrade package, but at the moment, that pkg only focus on language feature, not DOM apis. what should we do?